src/sentry/static/sentry/less/shared-components.less

```diff
@@ -954,55 +954,6 @@ table.integrations {
 * Filter Grid
 * ============================================================================
 */
-
-.filter-grid-item {
```

Comment:
Will this not impact legacy pages (before switch)?

Reply:
Project settings are all using shared views, this would update the inbound filters views for everyone.

Reply:
Okay. Will these layouts look okay in the old layout? @ckj